### ISSUE TITLE
My training results are not as good as those in your paper

### ISSUE THREAD

**Issue by lnSong** [noahzn/Lite-Mono#15]
Hello, I would like to ask why I trained your code from scratch, and the results differ significantly from yours? I used the default hyperparameter in your code. Have you used monodepth2's automasking loss? Thank you very much for being able to open source, which has been very helpful to me!

**Comment by noahzn** [comment]
Hi, could you give me more information? The automasking loss is used because I didn't set `--disable_automasking`. 
I would suggest you take a look at this [issue](https://github.com/noahzn/Lite-Mono/issues/7) and this [issue](https://github.com/noahzn/Lite-Mono/issues/4).

**Comment by lnSong** [comment]
Thank you for your reply. My previous training was in. png format, but now it has been changed to. jpg format and the batch size has been set to 12. The results have been improved

**Comment by lnSong** [comment]
Thank you for your reply. My previous training was in. png format, but now it has been changed to. jpg format and the batch size has been set to 12. The results have been improved.The previous training did not converge at all

**Comment by noahzn** [comment]
Good to know! Please close the issue if it is solved. Thanks.
